tests/cpp_api/performance_tests_with_depth/performance_tests_with_depth.cpp

```diff
@@ -316,7 +316,7 @@ int test(int test_case, const char* path, int rgb, int gpu, int width, int heigh
             for (int j = 0; j < batch_size; j++) {
                 tensor0 = input_image;
                 for (int k = 0; k < graph_depth; k++) {
-                    tensor0 = rocalWarpAffineFixed(handle, tensor0, true, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5);
+                    tensor0 = rocalWarpAffineFixed(handle, tensor0, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, true);
```

Comment:
why are the parameters changing here?

Reply:
The arguments in this particular test file was not in the right order leading to [-Wliteral-conversion] warnings, Hence it has been fixed in this PR